Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -60,8 +60,27 @@ def search_alerts(self, last_modified, severity, category, vendor, time_from, ti
         response = self.ms_client.http_request(method='GET', url_suffix=cmd_url, params=params)
         return response
 
-    def get_alert_details(self, alert_id):
-        cmd_url = f'security/alerts/{alert_id}'
+    def search_alerts_v2(self, last_modified, severity, time_from, time_to, filter_query):
```

Comment:
`search_alerts_v2` looks similar to `search_alerts` please try to change the `search_alerts` method by adding the parameter `use_alerts_v2=False` so it works for v1 and v2